### DISCUSSION TITLE
Break attribute validation

### DISCUSSION THREAD

**Discussion by lanrenbulan** [laravel/framework#49016] [Ideas]
Each attribute can have multiple rules and can break before any rule.

```
$rules = [
     // Validation before `bail`, if there is an error, validation will not continue
      'code' => ['required', 'string', 'max:100', 'bail', 'unique']
]
```

We can specify the location of `bail`. If there is an error before, the verification will not continue. This can reduce one DB query.

**Comment by shaedrich** [comment]
Isn't that already possible? https://laravel.com/docs/10.x/validation#stopping-on-first-validation-failure

And if not, what are you suggesting compared to the current behavior of the `bail` rule?

**Reply by lanrenbulan** [reply]
It's not just that the validation is stopped when the first error occurs, but that the bail's position is determined if all the previous rules in it are completed successfully before the validation continues.
